src/lib/utils/retry-options.ts

```diff
@@ -0,0 +1,22 @@
+import { Options as RetryOptions } from 'p-retry';
+
+const defaultRetryOptions: RetryOptions = {
+	retries: 5,
+	factor: 2,
+	minTimeout: 500,
```

Comment:
If we go with `300`, the possible emits will happen at the following times
- [300, 600]
- [600, 1200]
- [900, 1800]
- [1200, 2000]
- [1500, 2000]

If we go with `500`
- [500, 1000]
- [1000, 2000]
- [1500, 2000]
- [2000, 2000]
- [2000, 2000]

When reading the `retry` docs, it looks like the emits are calculated individually. This means that in case of `300`, it's possible to the first retry happens after `600ms` (if random is `2`), and the second retry happens directly after that one (if random is `1`). Not sure about this behaviour though.

So looking at the times, I believe we should go with `300` here. What do you think?